src/ui/purchases-ui.svelte

```diff
@@ -102,9 +113,10 @@
 
   onDestroy(() => {
     if (!isInElement) {
-      document.documentElement.style.height = "auto";
-      document.body.style.height = "auto";
-      document.documentElement.style.overflow = "auto";
+      // Restore original styles
+      document.documentElement.style.height = originalHtmlHeight || "";
+      document.body.style.height = originalBodyHeight || "";
+      document.documentElement.style.overflow = originalHtmlOverflow || "";
```

Comment:
This way you can probaby just do

```suggestion
      document.documentElement.style.height = originalHtmlHeight;
      document.body.style.height = originalBodyHeight;
      document.documentElement.style.overflow = originalHtmlOverflow;
```

?